domains/certificates/Query.API/API/Configurations/MeasurementsSyncOptions.cs

```diff
@@ -17,6 +17,9 @@ public class MeasurementsSyncOptions
 
     [Required]
     public MeasurementsSyncerSleepType SleepType { get; set; }
+
+    [Required]
+    public int MinimumAgeBeforeIssuingInHours { get; set; }
```

Comment:
Now that this is configurable, what happens when we change this back or forth in time?

Add tests to verify that we handle when the configuration changes. And tests that make sure we do not miss any certificates.